pgbouncer-krb/entrypoint.sh

```diff
@@ -60,9 +61,10 @@ fi
 if [ ! -f ${PG_CONFIG_DIR}/pgbouncer.ini ]; then
   echo "Create pgbouncer config in ${PG_CONFIG_DIR}"
 
-# Config file is in �ini� format. Section names are between �[� and �]�.
-# Lines starting with �;� or �#� are taken as comments and ignored.
-# The characters �;� and �#� are not recognized when they appear later in the line.
+# Config file is in �ini� format. Section names are between �[� and �]�.
```

Comment:
What's up with these `¿ini¿` things? Shouldn't that be `"ini"` or something?

Reply:
@danielhoherd fixed.